### ISSUE TITLE
[BITV] 9.3.3.2/9.2 - Search input field "Search docs" lacks visual label and there is no search button or symbol to help identify it as search field. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/documentation#9739]
- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v9-n2

</details>
